src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsVuapMercury.java

```diff
@@ -0,0 +1,41 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.core.annotations.NetworkTrafficDump;
+import com.wikia.webdriver.common.core.url.Page;
+import com.wikia.webdriver.common.dataprovider.ads.AdsDataProvider;
+import com.wikia.webdriver.common.templates.mobile.MobileTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.ad.VUAP;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsBaseObject;
+import org.testng.annotations.Test;
+
+import java.util.Map;
+
+@Test(
+        groups = "AdsVuapMercury"
+)
+public class TestAdsVuapMercury extends MobileTestTemplate {
+
+    private static final String FANDOM_URL = "http://www.wikia.com/fandom";
+    private static final String URL = "pagead2.googlesyndication.com/pagead/gen_204?rt";
```

Comment:
Per our offline discussion: let's remove this and focus on `ads?` requests with correct params: `output=vast` and correct position in `cust_params`.